packages/contracts-bedrock/src/L1/L1ERC721Bridge.sol

```diff
@@ -23,19 +23,32 @@ contract L1ERC721Bridge is ERC721Bridge, ISemver {
     SuperchainConfig public superchainConfig;
 
     /// @notice Semantic version.
-    /// @custom:semver 2.0.0
-    string public constant version = "2.0.0";
+    /// @custom:semver 2.1.0
+    string public constant version = "2.1.0";
 
     /// @notice Constructs the L1ERC721Bridge contract.
-    /// @param _messenger   Address of the CrossDomainMessenger on this network.
-    constructor(address _messenger) ERC721Bridge(_messenger, Predeploys.L2_ERC721_BRIDGE) {
-        initialize(SuperchainConfig(address(0)));
+    constructor() ERC721Bridge() {
+        initialize({
+            _messenger: CrossDomainMessenger(address(0)),
+            _otherBridge: Predeploys.L2_ERC721_BRIDGE,
+            _superchainConfig: SuperchainConfig(address(0))
+        });
     }
 
     /// @notice Initializes the contract.
-    /// @param _superchainConfig Address of the SuperchainConfig contract on this network.
-    function initialize(SuperchainConfig _superchainConfig) public initializer {
+    /// @param _messenger   Contract of the CrossDomainMessenger on this network.
+    /// @param _otherBridge Address of the ERC721 bridge on the other network.
+    /// @param _superchainConfig Contract of the SuperchainConfig contract on this network.
+    function initialize(
+        CrossDomainMessenger _messenger,
+        address _otherBridge,
+        SuperchainConfig _superchainConfig
+    )
+        public
+        initializer
+    {
         superchainConfig = _superchainConfig;
+        __ERC721Bridge_init(_messenger, _otherBridge);
```

Comment:
nit: can you update the call to `__ERC721Bridge_init` to using kwargs style syntax, it is easier to review. Will guarantee the args are in the correct order